twopane-navigation/src/components/paneRenderer/PaneRenderer.tsx

```diff
@@ -73,12 +73,23 @@ const PaneRenderer = (props: IPaneRendererProps) => {
         
     }
 
-// TODO MOVE TO ONE RENDER
+    const isAccessible = (component: IPaneComponent) => {
+        const isLastPaneOne = component.pane === paneType.ONE && paneComponents.filter(x => x.pane === paneType.ONE).pop()?.key === component.key;
+        const isLastPaneTwo = component.pane === paneType.TWO && paneComponents.filter(x => x.pane === paneType.TWO).pop()?.key === component.key;
```

Comment:
you can save one search thru the component list by first having a filtered list of items that match the component key.

if paneComponents always has max 2 elements, the disregard